dipy/stats/analysis.py

```diff
@@ -147,7 +112,13 @@ def dti_measures(bundle, metric, dt, pname, bname, subject, group, ind, dir):
             path of output directory
     """
 
-    dt["bundle"] = []
+    if group == 'patient':
```

Comment:
This is a good point.

Reply:
I agree with your point that it should be generic and not fixed to have two groups with specific names (control and patient). For some other project, we might get more than two groups with different names.

For now, we are only supporting comparisons between two groups (control vs patient). The whole pipeline is written in such a way that we need two groups control and patient.

For future work, I believe, changing this code and making it more generic would not be very difficult. Making changes in this PR might take longer and disturb the flow. We can make changes to support more groups in the next PR.

Reply:
Sounds like a plan. So, can you:
- update the docstring to avoid the confusion
- raise an exception or an warning if this not 'patient' or 'control'
- use `group.lower() == patient` in case someone use uppercase.

Reply:
now it takes input direct `group_id`